resources/profiles/AnkerMake.ini

```diff
@@ -235,47 +207,47 @@ filament_type = PLA
 filament_density = 1.24
 filament_cost = 20
 first_layer_bed_temperature = 60
-first_layer_temperature = 210
+first_layer_temperature = 230
 fan_always_on = 1
 max_fan_speed = 100
 min_fan_speed = 100
 bridge_fan_speed = 100
-disable_fan_first_layers = 2
-temperature = 205
+disable_fan_first_layers = 1
+temperature = 200
 
 [filament:*PLA+*]
 inherits = *common*
 bed_temperature = 60
 fan_below_layer_time = 100
 filament_colour = #DDDDDD
-filament_type = PLA
+filament_type = PLA+
 filament_density = 1.24
 filament_cost = 20
 first_layer_bed_temperature = 60
-first_layer_temperature = 220
+first_layer_temperature = 230
 fan_always_on = 1
 max_fan_speed = 100
 min_fan_speed = 100
 bridge_fan_speed = 100
-disable_fan_first_layers = 2
-temperature = 210
+disable_fan_first_layers = 1
+temperature = 200
 
 [filament:*PET*]
 inherits = *common*
-bed_temperature = 70
+bed_temperature = 80
 disable_fan_first_layers = 2
 fan_below_layer_time = 20
 filament_colour = #DDDDDD
 filament_type = PETG
 filament_density = 1.27
 filament_cost = 30
-first_layer_bed_temperature = 70
-first_layer_temperature = 240
+first_layer_bed_temperature = 80
+first_layer_temperature = 260
 fan_always_on = 1
 max_fan_speed = 50
 min_fan_speed = 50
 bridge_fan_speed = 100
-temperature = 240
+temperature = 260
```

Comment:
@ankermake I would reduce this to 255 as you are pushing the limit of the PTFE tube for off-gassing since you do not have an all-metal hotend.